api/swagger.yml

```diff
@@ -5240,6 +5240,18 @@ paths:
         required: true
         schema:
           type: string
+      - in: query
+        name: mode
+        description: |
+          physical - Default. Uses copy of new object's physical address.
+          logical - Creates an entry point to the same physical address.
+          *EXPERIMENTAL*
```

Comment:
```suggestion
        Determines how the object is copied:
          - physical (default): Creates a new copy of the object in both lakeFS and the underlying storage (e.g., S3). Results in two independent objects.
          - logical: Creates a new lakeFS entry that points to the same underlying storage object. Both lakeFS objects share the same physical data in storage (e.g., same S3 object).
          *EXPERIMENTAL*
```

Reply:
It might be worth mentioning that running this is only safe in two modes
- the physical object was created less than 6 hours ago
- We know for sure that GC isn't currently running 
If GC is running and the file was created more than 6 hours ago, there are some edge cases that GC can remove the file even though it still has one pointer